app/views/red_line_boundary_change_validation_requests/show.html.erb

```diff
@@ -21,14 +21,14 @@
     <p class="govuk-body">
       <%= @red_line_boundary_change_validation_request.reason  %>
     </p>
-    <%= render "shared/location_map", locals: { geojson: @planning_application.boundary_geojson } %>
+    <%= render "shared/location_map", locals: { geojson: @red_line_boundary_change_validation_request.original_geojson } %>
```

Comment:
I'm getting this error for the shared/location_map view:

![image](https://user-images.githubusercontent.com/9452321/146779009-7ed405ec-c2ea-4a52-9f7d-0eb4dfdcf99e.png)

Reply:
hm would like to see your db for this

Reply:
seems to have resolved itself